src/modules/products/screens/detail-products/details-screen.tsx

```diff
@@ -0,0 +1,105 @@
+};
+const availableItems = 12;
+
+const images = [
+  require('../../../../ui/assets/images/product-details/image-5.png'),
+  require('../../../../ui/assets/images/product-details/image-6.png'),
+  require('../../../../ui/assets/images/product-details/image-5.png'),
+];
+
+export const DetailsScreen = () => {
+  const [selectedImage, setSelectedImage] = useState(0);
+  const navigation = useNavigation();
+
+  const changePrincipalImage = (index: number) => {
+    setSelectedImage(index);
+  };
+
+  const handleBackToHome = () => {
+    navigation.navigate('Tab');
+  };
+
+  return (
+    <SafeAreaView className="flex-1 bg-gray-100">
+      <ScrollView>
+        <View>
+          <Image
+            style={{ width: '100%' }}
+            source={require('../../../../ui/assets/images/logo-black-market/Header.png')}
+          />
+        </View>
+        <View className="px-4 py-3">
+          <View className="flex-row justify-between">
+            <View className="w-1/4 items-center justify-center rounded-md bg-[#559F21]">
+              <Text variant="body1" className="text-white">
+                {item.state}
+              </Text>
+            </View>
+            <TouchableOpacity className="mr-4" onPress={handleBackToHome}>
+              <Text className="text-2xl font-bold text-red-700">X</Text>
+            </TouchableOpacity>
+          </View>
+          <Text variant="h3">{item.name}</Text>
+          <Text variant="h3" className="font-light">
+            {common.details_screen.byManufacturer} {item.manufactur}
+          </Text>
+          <Text variant="h3" className="font-medium">
+            {common.labels.dolarSign} {item.price}
+          </Text>
+          <Image source={images[selectedImage]} className="my-4 h-[300px] w-full rounded-lg" />
+          <ScrollView horizontal showsHorizontalScrollIndicator={false} className="mb-4">
+            {images.map((image, index) => (
+              <TouchableOpacity key={index} onPress={() => changePrincipalImage(index)}>
+                <Image source={image} className="w-28 h-28 mr-4 rounded-md" />
+              </TouchableOpacity>
+            ))}
+          </ScrollView>
+          <View className="mb-4 flex-row items-center justify-between">
+            <View className="items-center">
+              <Text variant="body1" className="mb-3 mr-2 font-bold">
+                {common.details_screen.quantity}
+              </Text>
+              <DropdownComponent quantity={availableItems} />
+            </View>
+            <View className="w-2/3 items-center">
+              <Text variant="body1" className="mb-2 mr-2 font-bold">
+                {common.details_screen.availability} {availableItems} {common.details_screen.items}
+              </Text>
+              <TouchableOpacity className="w-full items-center rounded-md bg-slate-900 p-3">
+                <Text className="text-lg font-bold text-white">{common.labels.buy}</Text>
+              </TouchableOpacity>
```

Comment:
Why not making use of the Button component you've already built?

By reusing components:
1. You save time each time you need to add a new button for example, or a new input, etc. 
2. You reduce the amount of errors you might have on paddings or size not being pixel perfect (because you've already designed them once and that's enough)
3. If a design change is introduced on one of the components it's easier to update, instead of going through all screens updating the component, you just update it once. 
4. It makes code more readable

Reply:
Done